tetrateci/create_istio_release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir /tmp/istio-release
+sudo gem install fpm
+sudo apt-get install go-bindata -y
+envsubst < ./tetrateci/manifest.yaml.in > manifest.yaml
+git clone https://github.com/istio/release-builder --depth=1
```

Comment:
I think it is better to clone this outside of the current repo. WDYT?

Reply:
I think I would just remove the checkout

Reply:
Oh wait, we don't get the manifest if we dont fetch the repo